src/main/java/com/ksm/domino/cli/command/project/ProjectCopy.java

```diff
@@ -61,14 +61,14 @@ public void execute() throws Exception {
         DominoProjectsApiCopyGitRequest gitObject = new DominoProjectsApiCopyGitRequest();
         gitObject.setCredentialId(credentialId);
         gitObject.setImportedGitReposCredentialId(importedCredentialId);
-        
+
         // - enforce link xor new repo
         if (linkRepoUrl.isPresent() && repoName.isEmpty() && repoOwner.isEmpty()) {
             gitObject.setLinkSpec(new DominoProjectsApiLinkedGitRepoMetadata().uri(linkRepoUrl.get()));
         } else if (repoName.isPresent()) {
             Validate.notBlank(repoOwner.orElse(null), "Parameter 'repoOwner' is required if 'repoName' is specified");
             DominoProjectsApiCopiedGitRepoMetadata copySpec = new DominoProjectsApiCopiedGitRepoMetadata();
-            copySpec.setIsPrivate(true);
+            copySpec.setVisibility(DominoProjectsApiCopiedGitRepoMetadata.VisibilityEnum.PRIVATE);
```

Comment:
Just to note this specific change makes it incompatible with 5.5.1, as `isPrivate` is required in 5.5.1 and was change to required field `visibility` for 5.7.0. We would need to add both fields to the base domino-java-client to fix that, or we may want to track compatible versions in some way